angular-client/components/project.js

```diff
@@ -23,21 +23,22 @@ app.controller('project' , function ($scope,$http ) {
 		data : data
 	}
 	$http(post).then(function () {
-		console.log('success')
+		console.log('success');
 	},function () {
-		console.log('error')
+		console.log('error');
 	})
  	
  }
- get()
+ get();
  $scope.addproject = function () {
  	console.log($scope.projectName+" "+$scope.projectDesc)
  	post({
  		
  			projectName  : $scope.projectName,
  			projectDisc : $scope.projectDesc
- 	})
-get()
- }
- 
+ 	});
+ 	
+get();
```

Comment:
You need to fix this indentation. This should be indented at the same level as the console.log statement and the post call.

Reply:
Also, I can't comment on it, since it's not technically part of the PR, but your declaration of the "get" function above is a bit confusing. I'm reproducing it below for reference.

```
 	var get = function () {
 	var get={
 		method:"GET",
 		url : '/project'
 	}
 		$http(get).then(function (data) {
 			$scope.getter = data.data
 		},function () {
 			console.log('error')
 		})
  }
```

First, you're declaring a variable called get right inside of your get function, which also uses the `var get` syntax. This makes the code hard to follow, since, if I read the word 'get' somewhere, I'm not sure what it's referring to. I'd name the second get, which holds information on your request, something like `requestDetails`, since this describes what the variable is for. Also, the indentation for the whole function is off. Your second get variable should be declared at the same indentation level as the $http function call, since both of these actions are happening _inside_ of the get function. Any time we're putting actions in as part of a function, we indent them further right than the function declaration itself.

Furthermore, the end bracket '}' of your function should be indented at the same level as the beginning of your function. Example below.
```
function addOne(number) {
  return number + 1;
}
```